data/mohit1607.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "MOHIT ASHLIYA",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast with taste in React!",
+  "avatar": "https://github.com/mohit1607.png",
+  "tags": ["javascript", "typescript", "react.js", "node.js", "mongodb", "fullstack"],
+  "socials": [
+    { "icon": "twitter", "url": "https://twitter.com/ashliya_mohit" },
+    { "icon": "github", "url": "https://github.com/mohit1607" }
```

Comment:
These are not icons, please use icons available https://linkfree.eddiehub.io/icons